src/test/java/microsoft/exchange/webservices/data/core/EwsUtilitiesTest.java

```diff
@@ -35,14 +34,4 @@
   public void testGetBuildVersion() {
     Assert.assertEquals("Build version must be 0s", "0.0.0.0", EwsUtilities.getBuildVersion());
```

Comment:
Completely agree, but I think it will be better to create separate ticket for this issue.

PS: We could also use some Maven plugin to do it.